compiler/src/main/kotlin/io/spine/protodata/ProtoSourceFileProjection.kt

```diff
@@ -35,49 +36,49 @@ import io.spine.server.projection.Projection
 public class ProtoSourceFileProjection
     : Projection<FilePath, ProtobufSourceFile, ProtobufSourceFile.Builder>() {
 
-    @Subscribe(external = true)
-    internal fun on(e: FileEntered) {
+    @Subscribe
+    internal fun on(@External e: FileEntered) {
         builder()
```

Comment:
Please try the `TransactionalEntity.update()` extension recently introduced in `core-java` (see `TransactionalEntityExtensions.kt`.

Reply:
Right now, ProtoData uses Spine v1.7.0. We'll migrate to v2.0 as soon as `bootstrap` is ready. And to release `bootstrap`, we need to release a few Spine libraries first (such as `web` and `gcloud`).